### ISSUE TITLE
analytics code

### ISSUE THREAD

**Issue by paolobenve** [paolobenve/photofloat#36]
implemente an options to include google analytics (or other service) code in page via php

**Comment by paolobenve** [comment]
implemented in development branch, testing needed

**Comment by paolobenve** [comment]
it's in master, @pmetras, wouldn't you test it?

**Comment by pmetras** [comment]
I'll do it soon.

**Comment by pmetras** [comment]
That's correct for me. When setting the Piwik or Google Analytics options and running again the scanner, I can see them in the source code of the web page.
